modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -25,7 +25,7 @@ trap nagios_passive EXIT
 
 # Backup mongodb
 TIME="$(date +%s)"
-/usr/bin/mongodump -h $BACKUP_NODE -o $BACKUP_DIR --quiet
+/usr/bin/mongodump -h $BACKUP_NODE -o $BACKUP_DIR &> /dev/null
```

Comment:
Why are we suppressing this? I imagine it'd be handy to have processed output sent to the normal logs address.